core/include/msg_bus.h

```diff
@@ -103,6 +103,7 @@ static inline bool msg_is_from_bus(const msg_bus_t *bus, const msg_t *msg)
  * posted on the bus.
  *
  * Events can be received with @ref msg_receive.
+ * <b>The contents of the received message must not be modified.</b>
```

Comment:
We typically use either doxygen or markdown syntax in the doc:
```suggestion
 * **The contents of the received message must not be modified.**
```